### PR TITLE
Add wayvncctl get-outputs, disconnect-client, and wayvnc-exit commands

### PR DESCRIPTION
**Pull request by lack**
Adds `get-outputs`, `disconnect-client`, and `wayvnc-exit` commands, plus a `capture-changed` event.

I have read and understood CONTRIBUTING.md and its associated documents.

### CODE REVIEW

File: src/ctl-server.c [hunk 1]
```diff
@@ -306,6 +303,7 @@ static struct cmd* parse_command(struct jsonipc_request* ipc,
 						ipc->method),
 					"commands", list_allowed_commands()));
 	}
+	cmd->type = cmd_type;
```
Comment (any1) [inline src/ctl-server.c:352]: If `cmd_type == CMD_UNKNOWN`, you will have a null-dereference here.

File: wayvnc.scd [hunk 2]
```diff
@@ -262,6 +262,16 @@ Parameters:
 *username=...*
 	The username used to authenticate this client. May be null.
 
+_CAPTURE_CHANGED_
+
+The *capture-changed* event is sent when the currently caotured output
```
Comment (any1) [inline wayvnc.scd:271]: caotured

**Comment by lack** [comment]
@any1 Thanks, and good catches!  Fixed.

**Comment by any1** [comment]
Thanks!

**Pull request merged**